src/pages/home/Home.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import Navbar from "../../component/navbar/Navbar";
+import TodoForm from "../../component/form/Form";
+import TodoList from "../../component/todoList/homeTodoList/HomeTodoList";
+
+function Home() {
+  const [formState, changeFormStates] = useState({
+    title: "",
+    date: new Date(),
+    todoItem: [],
+    buttonStatus: "",
+  });
+
+  const handleSubmit = (e) => {
```

Comment:
make this to a custom hook. as per my blogs. all this code should not be here at all

Reply:
yes sir